### PR TITLE
♻️color stories

### PR DESCRIPTION
**Pull request by bdolly**
Super charges color stories white shade values in both hex and rgba as well as full table of acceptable color contrasts valus

### CODE REVIEW

File: stories/Colors/Colors.jsx [hunk 1]
```diff
@@ -1,34 +1,101 @@
...
+      rgb(
+      {color.rgba.slice(0, 3).join(',')})
+    </span>
+  </small>
+);
 
 /**
  * A basic square swatch to display a color
  */
 function Swatch(props) {
```
Comment (dankolbman) [inline stories/Colors/Colors.jsx]: Is it worth pulling some of this out into other components to tidy up a little?

File: stories/Colors/Colors.jsx [hunk 2]
```diff
@@ -1,34 +1,101 @@
...
+      <div className="group w-full my-2 rounded-sm ">
+        <h5 className="text-center">A11y Contrasts</h5>
+        <small className={`block w-full h-8  p-2 bg-${colorData.key} text-${idealTextColor}`}>
+          <span className="inline-block leading-normal float-left">
+            text-
+            {idealTextColor[0]}{' '}
+          </span>
+          <span className="inline-block leading-normal float-right">{idealTextColor[1]}</span>
+        </small>
+        <div className="hidden group-hover:block">
```
Comment (dankolbman) [inline stories/Colors/Colors.jsx]: Could this be shown by default? The hover state is pretty drastic.

File: package.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
   },
   "scripts": {
     "tailwind:css": "tailwind build src/css/tailwind.src.css -c  tailwind.js -o src/css/tailwind.css",
-    "storybook": "start-storybook -p 9001 -c .storybook",
+    "storybook": "NODE_ENV=testing start-storybook -p 9001 -c .storybook",
```
Comment (dankolbman) [inline package.json]: These changes should be done in a different pr.

File: stories/Colors/Colors.jsx (hunk 1 above)
Comment (bdolly) [reply stories/Colors/Colors.jsx]: eh, I debated that but the color swatch isn't meant to be export for use in other apps so I think it's ok

File: stories/Colors/Colors.jsx (hunk 2 above)
Comment (bdolly) [reply stories/Colors/Colors.jsx]: I felt like showing it initially cluttered the display and pulled the focus away from the color definitions

**Comment by bdolly** [comment]
@dankolbman are we good to merge this?

**Comment by dankolbman** [comment]
Can you rebase and see if it fixes the deploy?

**Comment by bdolly** [comment]
@dankolbman checks pass but getting a babelhelpers error on netlify. Is that due to storybook not being built with the proper NODE_ENV ?
